Samples/Caching/Caching.csproj

```diff
@@ -11,6 +11,7 @@
     </ItemGroup>
 
     <ItemGroup>
+        <ProjectReference Include="..\..\Backend\Remora.Discord.Caching.Redis\Remora.Discord.Caching.Redis.csproj" />
```

Comment:
Is there a reason the caching sample now uses Redis instead of normal memory caching? A separate redis project might be more appropriate.

Reply:
I believe this was a by-product of migrating things to a new project.